qiskit_alice_bob_provider/remote/display.py

```diff
@@ -33,7 +33,7 @@ def _new_line_ipython() -> None:
 
 
 def _display_ipython(text: str) -> None:
-    current_label.value = f'[{datetime.now().strftime("%H:%M:%S")}] {text}'
+    current_label.value = f"""[{datetime.now().strftime('%H:%M:%S')}] {text}"""
```

Comment:
I don't understand how this change is related to Python 3.12 support? Is it purely a style change? If yes I don't see the point as we use single quotes everywhere in the code.

Reply:
I updated to `pylint` v3 and `flakes8-quotes` 3.4 to make them compatible with python 3.12. I remember I had some trouble with specific rules and I changed the quotes, but I can't reproduce it anymore. I think I just misunderstood their behavior, these changes are indeed not needed.